sssom/cli.py

```diff
@@ -425,7 +422,8 @@ def partition(inputs: List[str], output_directory: str):
         # logging.info(f'Example: {cdoc.mapping_set.mappings[0].subject_id}')
         # logging.info(f'Writing to {ofn}. Size={len(cdoc)}')
         msdf = to_mapping_set_dataframe(cdoc)
-        write_table(msdf, ofn)
+        with open(ofn, "w") as file:
+            write_table(msdf, file)
```

Comment:
as good 'ol mando says, this is the way.

Better for the function not to have to think about what kind of input this is - if you _must_, you can always write a wrapper around a function that interacts with files to make it interpret a path string in a certain way